opencodeblocks/graphics/blocks/block.py

```diff
@@ -94,49 +114,50 @@ def boundingRect(self) -> QRectF:
         """ Get the the block bounding box. """
         return QRectF(0, 0, self.width, self.height).normalized()
 
+    def setTitleGraphics(self, color: str, font: str,
+                         size: int, padding: float):
+        """ Set the title graphics.
+
+        Args:
+            color: title color.
+            font: title font.
+            size: title size.
+            padding: title padding.
+
+        """
+        self.title_widget.setMargin(int(padding))
+        self.title_widget.setStyleSheet("QLabel { color : %s; }" % color)
```

Comment:
Use f-string instead
```suggestion
        self.title_widget.setStyleSheet(f"QLabel \{ color : {color}\}")
```